pkgs/by-name/do/dolphin-emu/package.nix

```diff
@@ -149,21 +153,29 @@ stdenv.mkDerivation (finalAttrs: {
       ln -s $out/Applications/Dolphin.app/Contents/MacOS/Dolphin $out/bin
     '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${placeholder "out"}/bin/dolphin-emu-nogui";
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
```

Comment:
This build is 30+ minutes on a powerful builder, please revert to `testers.testVersion`

Reply:
Tripping this check for stable releases is generally not an issue.
But imagine you're overriding the package to use a special git revision or fork: that 30-minute setupback you'd get if you forget to also update `version` will give you a sour aftertaste. `passthru.tests` do get built by ofborg on every version bump, which is generally when you care about it